src/main/java/uk/gov/companieshouse/web/pps/security/WebSecurity.java

```diff
@@ -23,6 +23,14 @@ public SecurityFilterChain temporaryStartPageSecurityFilterChain(HttpSecurity ht
         return http.build();
     }
 
+    @Bean
+    @Order(1)
```

Comment:
In line 19, there is Order (1), and it is duplicate in this line. It may need to reorder the numbers

Reply:
Thanks, I'm aware it is a duplicate Order(1), but felt it was ok to keep the same as I didn't notice an issue with the WebSecurity.

Reply:
Resolved in commit faa7c6a76e6b70a4bfc69a96f3005666b12059ed - also applied recent changes to WebSecurity config as seen in [#156](https://github.com/companieshouse/penalty-payment-web/pull/156)